packages/kbn-es-query/src/kuery/ast/ast.js

```diff
@@ -79,3 +79,7 @@ export function doesKueryExpressionHaveLuceneSyntaxError(expression) {
     return (e.message.startsWith('Lucene'));
   }
 }
+
+export function buildKueryNode(functionName, ...functionArgs) {
```

Comment:
`nodeTypes.function.buildNode` is exported from the top level of this module, so could whatever code that's using this function use `nodeTypes.function.buildNode` directly instead? I think this utility function could be confusing because it's called `buildKueryNode`, yet it only builds function nodes.

Reply:
@Bargs I know we talk over zoom, but I wanted to let you know that you can look at my last commit to see the way that you wanted it.